docs/source/user_guide/supported_features.md

```diff
@@ -0,0 +1,20 @@
+# Supported Features
+
+| Feature                       | V0 | V1 |
+|-------------------------------|----|----|
+| Quantization                  |    |✅︎  |
+| LoRA                          |⚠️  |    |
+| Tensor Parallel               |✅︎  |✅︎  |
+| Prompt Adapter                |✅︎  |✅︎  |
```

Comment:
I could believe this is supported since it should be applying updates to the prompt outside the model code, but I don't have a quick way to test it atm